omod/pom.xml

```diff
@@ -16,6 +16,11 @@
 			<artifactId>${project.parent.artifactId}-api</artifactId>
 			<version>${project.parent.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>${project.parent.groupId}</groupId>
+			<artifactId>${project.parent.artifactId}-api-1.10</artifactId>
```

Comment:
Do you need to add this dependency?

Reply:
Without this dependency, the drug order filter doesn't work. because it's in the 1.1 module and was missing in the depedencies.